<?xml version="1.0"?>
<launch>
 <!-- change this next line to include the launch file for your robot's hardware drivers: -->

	<node pkg="rosserial_python" type="serial_node.py" name="serial_node">
	    <param name="port" value="/dev/mega_base"/>
	    <param name="baud" value="1000000"/></node>

  <!-- urdf xml robot description loaded on the Parameter Server-->
	<param name="robot_description" command="$(find xacro)/xacro '$(find rugby)/modeldescription/urdf/rugby.xacro'" />
	
	<!-- Robot parameters -->
	<param name="wheelSeparation" value="0.245"></param>
	<param name="wheelRadius" value="0.0475"></param>
	<param name="rate" value="10"></param>  
	
	<!-- Starting Joint state publisher node which will publish the joint values -->
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	<!-- Starting robot state publisher -->
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher">
		<param name="publish_frequency" type="double" value="50.0"></param></node>
		
	<!-- Starting twist to motor and diff_tf nodes -->
	<node pkg="rugby" type="mybot_base_controller_v1_rugby" name="mybot_base_controller_v1_rugby" output="screen"/>
 	<node pkg="rugby" type="mybot_odom_v1_rugby" name="mybot_odom_v1_rugby" output="screen"/>
 	 


</launch>